arcgis-ios-sdk-samples/Geometry/Spatial relationships/SpatialRelationshipsViewController.swift

```diff
@@ -119,89 +119,80 @@ class SpatialRelationshipsViewController: UIViewController, AGSGeoViewTouchDeleg
         //
         // Identify graphics overlay
         geoView.identify(graphicsOverlay, screenPoint: screenPoint, tolerance: 12, returnPopupsOnly: false, maximumResults: 1) { [weak self]  (result) in
-            //
-            // Make sure self is around
-            guard let strongSelf = self else {
```

Comment:
`self` should still be unwrapped to a `strong` reference here since we're referencing it more than once in the same scope.

Reply:
👍